src/kibana/components/courier/saved_object/saved_object.js

```diff
@@ -265,6 +269,24 @@ define(function (require) {
         });
       };
 
+      self.export = function () {
+        return {
+          _id: this.id,
+          _type: type,
+          _source: this._source
+        };
+      };
+
+      self.import = function (result) {
```

Comment:
We talked about this in person, but the meaning of this method is a bit confusing. Might be better as a static method or a call to `#saveSource`.